charts/schulportal-client/templates/deployment.yaml

```diff
@@ -21,7 +21,9 @@ spec:
           ports:
             - name: http
               containerPort: {{ default 80 .Values.service.port }}
-              protocol: TCP 
+              protocol: TCP
           securityContext:
             allowPrivilegeEscalation: {{ default false .Values.allowPrivilegeEscalation }}
+          resources:
```

Comment:
https://github.com/dBildungsplattform/schulportal-client/actions/runs/8234697648/job/22570251264 gha not happy, syntax wrong